script.module.simpleeval/addon.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.simpleeval" name="Simple Eval" version="0.0.1" provider-name="SiLVO">
```

Comment:
author is the module author not yourself

Reply:
Will change this in next push